src/test/java/de/bonndan/nivio/input/dto/InputFormatHandlerNivioTest.java

```diff
@@ -87,16 +93,35 @@ public void readServiceAndInfra() {
         assertEquals("auth", web.getLabel("traefik.http.routers.router0.middlewares"));
     }
 
+
     @Test
-    public void readIngress() {
+    public void mergeGroups() {
 
         SourceReference file = new SourceReference(getRootPath() + "/src/test/resources/example/services/dashboard.yml");
 
-        List<ItemDescription> services = descriptionFactory.getDescriptions(file, null);
-        ItemDescription service = services.get(0);
-        assertEquals(Item.LAYER_INGRESS, service.getGroup());
-        assertEquals("Keycloak SSO", service.getName());
-        assertEquals("keycloak", service.getIdentifier());
+        //when
+        descriptionFactory.applyData(file, null, landscapeDescription);
+
+        //then
+        Map<String, GroupDescription> groups = landscapeDescription.getGroups();
+        assertThat(groups).isNotEmpty();
+
+        GroupDescription ingress = groups.get("ingress");
```

Comment:
@mrgamedev07 here we test for a new group definition that comes from a referenced file